src/components/CommentsList.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState, useEffect } from "react";
+import { fetchComments } from "../api";
+import CommentCard from "./CommentCard";
+import Lottie from "lottie-react";
+import loadingAnimation from "../assets/loadingAnimation.json";
+
+function Comments({ articleId }) {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetchComments(articleId)
+      .then((comments) => {
+        setComments(comments);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        setIsLoading(false);
+        setIsError(true);
+      });
+  }, []);
+
+  if (isLoading) {
+    return (
+      <Lottie animationData={loadingAnimation} className="loading-animation" />
```

Comment:
second time you've used this now. I think you're ready to warrant a `<Loading />` component :)